### DISCUSSION TITLE
How to write deploy.yml

### DISCUSSION THREAD

**Discussion by nadveos** [caprover/caprover#1937] [Q&A] [answered]
hi there, i m new in dev things and gb actions. That cade its correct to integrate gb action with caprover? thanks in advance`name: Flutter Web

on:
  push:
    branches:
      - master
      
jobs:
  build:
    runs-on: ubuntu-latest
    
    steps:
	  - uses: actions/checkout@v3
           - uses: subosito/flutter-action@v2
      with:
        channel: 'stable'
    - run: flutter pub get
    - run: flutter test
    - run: flutter build web --web-renderer canvaskit
    - name: Build and push Docker Image
      uses: docker/build-push-action@v5
      with:
        context: .
        file: ./Dockerfile
        push: true
        

    - name: Deploy Image to CapRrover
      uses: caprover/deploy-from-github@v1.1.2
      with:
        server: "${{ secrets.CAPROVER_SERVER }}"
        app: "${{ secrets.APP_NAME }}"
        token: "${{ secrets.APP_TOKEN }}"
        image: ${{ env.IMAGE_URL }}`
Thanks in advance

**Comment by githubsaturn** [comment, marked as answer]
No, it's not correct. Please look at the official docs here:
https://github.com/caprover/deploy-from-github#example-1---deploy-using-image
